tests/libcxx/expected_results.txt

```diff
@@ -182,6 +182,9 @@ std/language.support/support.limits/support.limits.general/version.version.compi
 # libc++ incorrectly implements `layout_stride::mapping<E>::is_exhaustive()`
 std/containers/views/mdspan/layout_stride/is_exhaustive_corner_case.pass.cpp FAIL
 
+# libc++ triggers UB in its tests by specializing `common_reference`
+std/iterators/iterator.requirements/iterator.concepts/iterator.concept.readable/indirectly_readable.compile.pass.cpp FAIL

```

Comment:
```
Unexpectedly Passed Tests (1):
  libc++ :: std/iterators/iterator.requirements/iterator.concepts/iterator.concept.readable/indirectly_readable.compile.pass.cpp:2
```

This means the test passed for `clang-cl` but failed for `cl`.

To make CI happy iun this case, need this instead:

```suggestion
std/iterators/iterator.requirements/iterator.concepts/iterator.concept.readable/indirectly_readable.compile.pass.cpp:0 FAIL
std/iterators/iterator.requirements/iterator.concepts/iterator.concept.readable/indirectly_readable.compile.pass.cpp:1 FAIL
```